packages/@cdk-cosmos/core/src/ecs-solar-system.ts

```diff
@@ -33,7 +33,7 @@ export interface IEcsSolarSystemCore extends ISolarSystemCore {
   alb: IApplicationLoadBalancer;
   httpListener: IApplicationListener;
   httpInternalListener: IApplicationListener;
-  // HttpsListener: IApplicationListener;
+  httpsListener: IApplicationListener;
```

Comment:
also this should be optional ? since its not required?

i think it should be required ?